libs/admin/src/lib/components/add-users/add-workgroup-user-dialog.component.scss

```
@@ -0,0 +1,13 @@
+.header {
+  padding: 20px 24px 0 24px;
+}
+
+.select {
+  width: 100%;
+}
+
+.actions {
+  display: flex;
+  justify-content: end;
```

Prefer `flex-start` and `flex-end` over `start` and `end` in flex layouts, as the latter two are not as widely supported.
```suggestion
  justify-content: flex-end;
```